Contest_API.md

```diff
@@ -330,6 +330,8 @@ access to.
 | [contest\_thaw](#modifying-contests)       | Control the contest's thaw time
 | [team\_submit](#modifying-submissions)     | Submit as a team
 | [team\_clar](#modifying-clarifications)    | Submit clarifications as a team
+| [judge\_clar](#modifying-clarifications)   | Submit clarifications as a judge
```

Comment:
I think we should merge the `team_clar` and `judge_clar` capabilities into one. I don't really see why you'd want to support one but not the other and they're not really different in complexity, except maybe that a judge may respond to an existing clarification and a team might not(?) This is for example different for the `submit_*` capabilities, where the admin or proxy one can override more attributes and/or have slightly different semantics.

Reply:
So one capability, but it says if your account type is team you can do X and if your account is judge you have slightly different access? Yes, I can do that, just wasn't sure that everyone would be willing to implement both at the same time - but I guess you could implement one and only advertise the capability via /access to that account type.

Reply:
PR updated, please re-review.